sky/utils/controller_utils.py

```diff
@@ -253,7 +254,7 @@ def _get_cloud_dependencies_installation_commands(
                 'pip list | grep cudo-compute > /dev/null 2>&1 || '
                 'pip install "cudo-compute>=0.1.10" > /dev/null 2>&1 && '
                 'wget https://download.cudo.org/compute/cudoctl-0.3.2-amd64.deb -O ~/cudoctl.deb > /dev/null 2>&1 && '  # pylint: disable=line-too-long
-                'sudo dpkg -i ~/cudoctl.deb > /dev/null 2>&1')
+                'sudo dpkg -i ~/cudoctl.deb 2>&1 > /dev/null')
```

Comment:
is there any specific reason we need to change this? Wondering if those two version works as the same.